flink-runtime/src/main/java/org/apache/flink/runtime/resourcemanager/ResourceManager.java

```diff
@@ -52,15 +58,28 @@
  * </ul>
  */
 public class ResourceManager extends RpcEndpoint<ResourceManagerGateway> {
-	private final Map<JobMasterGateway, InstanceID> jobMasterGateways;
+
+	private final Logger LOG = LoggerFactory.getLogger(getClass());
```

Comment:
There is a log field in RpcEndpoint, which is protected, why not use that instead?